troposphere/static/js/components/images/list/common/ImageListCard.react.js

```diff
@@ -46,29 +66,59 @@ export default React.createClass({
         let endDated;
         if (this.props.isEndDated) {
             endDated = (
-                <div style={{ position: "absolute", top: "10px", left: "0", background: "#F55A5A", display: "inline-block", padding: "5px 10px", color: "white" }}>
+                <div
+                    style={{
```

Comment:
Move this style into a `.style()` method. It'll clean up this div, and the span below it. Otherwise at a cursory glance the structure of the html cannot be discerned. I.e. let's remove clutter.